### PR TITLE
Add a conflict solver e2e test

### LINKED ISSUE #8178: E2E tests for conflict solver
Test navigation to conflict solver:
- Via Group menu
- Via previous and Next button
- Via "resolve issue" banner in Overview (-)
- Via "resolve issue" banner in cart (-)

This task has dependencies on #8382

### PR DESCRIPTION
**Pull request by Larisa-Staroverova**

### CODE REVIEW

File: projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration-overview.ts [hunk 1]
```diff
@@ -1,17 +1,92 @@
...
+const resolveIssuesLinkSelector =
+  'cx-configure-cart-entry button.cx-action-link';
 
+/**
+ * Navigates to the configured product overview page.
+ *
+ * @param configuratorType - Configuration type
+ * @param productId - Product ID
+ */
+export function goToConfigOverviewPage(configuratorType, productId) {
```
Comment (ChristophHi) [inline projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration-overview.ts]: I suggest to specify void return type here

File: projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration-overview.ts [hunk 2]
```diff
@@ -1,17 +1,92 @@
...
+  ).then(() => {
+    cy.get('.VariantConfigurationOverviewTemplate').should('be.visible');
+    this.isConfigOverviewPageDisplayed();
+  });
+}
+
+/**
+ * Verifies whether the product overview page is displayed.
+ */
 export function isConfigOverviewPageDisplayed() {
```
Comment (ChristophHi) [inline projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration-overview.ts:33]: Also please add return type

File: projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration.ts [hunk 3]
```diff
@@ -145,10 +251,33 @@ export function isAttributeValueNotDisplayed(
...
 export function getAttributeId(attributeName: string, uiType: string) {
   return `cx-config--${uiType}--${attributeName}`;
 }
 
+/**
+ * Retrieves the attribute label id.
+ *
+ * @param {string} attributeName - Attribute name
+ */
+export function getAttributeLabelId(attributeName: string) {
```
Comment (ChristophHi) [inline projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration.ts]: return type!

**Review by ChristophHi — CHANGES_REQUESTED** [review]
Could you specify the return types for the functions that you touched?
Especially for those functions that don't return void

File: projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration-overview.ts (hunk 1 above)
Comment (Larisa-Staroverova) [reply projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration-overview.ts]: Done

File: projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration-overview.ts (hunk 2 above)
Comment (Larisa-Staroverova) [reply projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration-overview.ts:33]: Done

File: projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration.ts (hunk 3 above)
Comment (Larisa-Staroverova) [reply projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration.ts]: Done

**Review by ChristophHi — APPROVED** [review]

**Pull request merged**